projects/plugins/jetpack/extensions/blocks/ai-assistant/edit.js

```diff
@@ -198,7 +220,7 @@ export default function AIAssistantEdit( { attributes, setAttributes, clientId }
 				ref={ aiControlRef }
 				content={ attributes.content }
 				contentIsLoaded={ contentIsLoaded }
-				getSuggestionFromOpenAI={ getSuggestionFromOpenAI }
+				getSuggestionFromOpenAI={ handleGetSuggestion }
```

Comment:
It stays the same, we just go through the `handleGetSuggestion` to make sure we focus the block, to allow the esc to happen.